SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -107,8 +107,20 @@ bool USpatialStatics::IsSpatialMultiWorkerEnabled(const UObject* WorldContextObj
 		return SpatialGDKSettings->bOverrideMultiWorker.GetValue();
 	}
 
-	const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
-	return WorldSettings != nullptr && WorldSettings->IsMultiWorkerEnabledInWorldSettings();
+	ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
+	if (WorldSettings != nullptr)
+	{
+		if (SpatialGDKSettings->OverrideMultiWorkerSettingsClass.IsSet())
+		{
+			FString OverrideMultiWorkerSettingsClass = SpatialGDKSettings->OverrideMultiWorkerSettingsClass.GetValue();
+			FSoftClassPath MultiWorkerSettingsSoftClassPath(OverrideMultiWorkerSettingsClass);
+			WorldSettings->MultiWorkerSettingsClass = MultiWorkerSettingsSoftClassPath.TryLoadClass<USpatialMultiWorkerSettings>();
```

Comment:
Only comment is that it seems odd to set WorldSettings->MultiWorkerSettingsClass in a function called IsSpatialMultiWorkerEnabled. I wonder if there is a nicer place to set MultiWorkerSettingsClass? It might be nice in the constructor of ASpatialWorldSettings. Obviously if ASpatialWorldSettings is constructed after reading cmd args.

Reply:
If set these parameters in the constructor of the ASpatialWorldSettings, the problem is that it might be overridden by the subsequent serialization.
As an explanation of this PR's description. I know set those settings in the boolean-returned function is not a better way. 
But because this work is blocking additional work and it follows an existing pattern I chose this solution.

Reply:
Cool that makes sense.

Reply:
Perhaps we could make a ticket to work out a nicer way of doing this in the future?

Reply:
Good recommendation!
Uniform the command line and settings configuration are not difficult. But cover all the test case include Zoning and Offloading is.
We had created a ticket:[UNR-4065](https://improbableio.atlassian.net/browse/UNR-4065)